editoast/src/views/infra/mod.rs

```diff
@@ -119,8 +123,17 @@ struct RefreshResponse {
 )]
 async fn refresh(
     app_state: State<AppState>,
+    Extension(authorizer): AuthorizerExt,
     Query(query_params): Query<RefreshQueryParams>,
 ) -> Result<Json<RefreshResponse>> {
+    let authorized = authorizer
+        .check_roles([BuiltinRole::InfraWrite].into())
```

Comment:
The frontend never uses this. This endpoint is used to force refresh. It's exclusively used by developers (ops)

Reply:
ok, Do you want me to delete the check role?

Reply:
Ideally, this endpoint should be available only for superusers.

Reply:
As discussed, we'll keep the check for now.